addons/web/static/src/views/fields/char/char_field.js

```diff
@@ -53,14 +54,23 @@ export class CharField extends Component {
         return this.props.record.fields[this.props.name].translate;
     }
     get formattedValue() {
-        return formatChar(this.props.record.data[this.props.name], {
+        const formattedValue = formatChar(this.props.record.data[this.props.name], {
             isPassword: this.props.isPassword,
         });
+
+        if (this.props.placeholderField) {
+            return formattedValue || this.placeholder;
```

Comment:
```suggestion
            return formattedValue || this.props.record.data[this.props.placeholderField];
```

`this.placeholder` contains the value of the placeholder field value if it is set, but also the static placeholder (set using `placeholder="..."` in the view) otherwise. 

I think we should maybe not show the static placeholder in the list view if the placeholder field isn't defined or is False, in order to not change the existing list view behavior.